tracer/src/Datadog.Trace/Configuration/ImmutableExporterSettings.cs

```diff
@@ -45,6 +46,7 @@ public ImmutableExporterSettings(ExporterSettings settings)
 
             PartialFlushEnabled = settings.PartialFlushEnabled;
             PartialFlushMinSpans = settings.PartialFlushMinSpans;
+            ValidationWarnings = settings.ValidationWarnings;
```

Comment:
I feel like we should call `ToList()` on this again, as otherwise we could run into concurrency issues if the user does something to the original `ExporterSettings` while we're doing the `foreach` in `TracerManager`

```suggestion
            ValidationWarnings = settings.ValidationWarnings.ToArray();
```